lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_generate_strings_file_from_code.rb

```diff
@@ -4,6 +4,7 @@ class IosGenerateStringsFileFromCodeAction < Action
       def self.run(params)
         files = files_matching(paths: params[:paths], exclude: params[:exclude])
         flags = [('-q' if params[:quiet]), ('-SwiftUI' if params[:swiftui])].compact
+        flags += Array(params[:routines]).flat_map { |s| ['-s', s] }
```

Comment:
I found that sequence of `s`s a bit confusing. How would you feel about a more verbose `routine`?

```suggestion
        flags += Array(params[:routines]).flat_map { |routine| ['-s', routine] }
```